environments/dev/dev.tfvars

```diff
@@ -0,0 +1 @@
+environment = "dev"
```

Comment:
this should be injected by the pipeline shouldn't need to be set, if you have no vars then just have an empty file